### PR TITLE
Updated GitHub references from GoogleCloudPlatform/forseti-security to forseti-security/forseti-security

### PR DESCRIPTION
**Pull request by red2k18**

### CODE REVIEW

**Comment by red2k18** [comment]
Fixes #3149

File: travis/jekyll_build.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ kill %1
 bundle exec htmlproofer --check-img-http --check-html \
 --internal-domains 'forsetisecurity.org' \
 --check-favicon --report-missing-names --report-script-embeds \
---url-ignore '/GoogleCloudPlatform/forseti-security/edit/,/forseti-security/forseti-security/edit/,/maxcdn.bootstrapcdn.com/,/d3js.org/' \
+--url-ignore '/forseti-security/forseti-security/edit/,/forseti-security/forseti-security/edit/,/maxcdn.bootstrapcdn.com/,/d3js.org/' \
```
Comment (dekuhn) [inline travis/jekyll_build.sh]: I don't think this reference needed to be updated. We can either should keep it or if everything is renamed we can just remove it.

Comment (gkowalski-google) [reply travis/jekyll_build.sh]: Agree, it can just be removed since the next URL to be ignored is the forseti-security URL.

Comment (red2k18) [reply travis/jekyll_build.sh]: Removed as everything else is updated.

Comment (gkowalski-google) [reply travis/jekyll_build.sh]: Do we will still need the other url ignores? I thought @dekuhn was just referencing the first url in that list.

**Pull request merged**